src/RetrySettings.php

```diff
@@ -489,6 +505,11 @@ public function getTotalTimeoutMillis()
         return $this->totalTimeoutMillis;
     }
 
+    public function getRetryFunction()
```

Comment:
Same as above - let's mark this as `@experimental`, as well as in the options array (although tags aren't possible there, so just write "This is Experimental" or something similar), and remove it from the private variable.